tests/lean/ppUnicode.lean

```diff
@@ -0,0 +1,3 @@
+#check fun a b c ↦  a + b + c + 1
+set_option pp.unicode.fun true in
+#check fun a b c => a + b + c + 1
```

Comment:
Would it be worth adding
```lean
set_option pp.unicode.fun true in
#check ∃ n : Nat, n ≤ 0
```
with expected output
```text
∃ n, n ≤ 0 : Prop
```
to make sure that the [issue with the previous implementation](https://github.com/leanprover/lean4/issues/1979) is never reintroduced?

Reply:
Done.